From 22832a0a15227e3fcc364b356247d8aeb9ce45b3 Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Sat, 27 Aug 2022 04:05:31 +0100
Subject: [PATCH 1/2] bootm: fix typo imape_comp -> image_comp

Chage variable name 'imape_comp' to the supposedly intended name
'image_comp'.

Signed-off-by: Daniel Golle <daniel@makrotopia.org>
---
 boot/bootm.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/boot/bootm.c
+++ b/boot/bootm.c
@@ -973,7 +973,7 @@ static int bootm_host_load_image(const v
 	int noffset;
 	ulong load_end;
 	uint8_t image_type;
-	uint8_t imape_comp;
+	uint8_t image_comp;
 	void *load_buf;
 	int ret;
 
@@ -991,20 +991,20 @@ static int bootm_host_load_image(const v
 		return -EINVAL;
 	}
 
-	if (fit_image_get_comp(fit, noffset, &imape_comp)) {
+	if (fit_image_get_comp(fit, noffset, &image_comp)) {
 		puts("Can't get image compression!\n");
 		return -EINVAL;
 	}
 
 	/* Allow the image to expand by a factor of 4, should be safe */
 	load_buf = malloc((1 << 20) + len * 4);
-	ret = image_decomp(imape_comp, 0, data, image_type, load_buf,
+	ret = image_decomp(image_comp, 0, data, image_type, load_buf,
 			   (void *)data, len, CONFIG_SYS_BOOTM_LEN,
 			   &load_end);
 	free(load_buf);
 
 	if (ret) {
-		ret = handle_decomp_error(imape_comp, load_end - 0, ret);
+		ret = handle_decomp_error(image_comp, load_end - 0, ret);
 		if (ret != BOOTM_ERR_UNIMPLEMENTED)
 			return ret;
 	}
